Makefile

```diff
@@ -123,12 +128,15 @@ simphony-openfoam:
 	@echo "Simphony OpenFoam plugin installed"
 
 simphony-jyu-lb:
-	pip install --upgrade git+https://github.com/simphony/simphony-jyulb.git@0.1.1#egg=jyu_engine
+	rm -Rf src/simphony-jyulb
+	git clone --branch 0.1.2 --depth 1 https://github.com/simphony/simphony-jyulb.git src/simphony-jyulb
+	git clone --branch master https://github.com/simphony/JYU-LB.git src/simphony-jyulb/JYU-LB
```

Comment:
in version 0.1.2 of simphony-jyulb, it is now required that jyulb repository be in its directory before running setup.py.  there is an issue related to it at  simphony/simphony-jyulb#21